README.md

```diff
@@ -38,7 +38,7 @@ To learn more about what we're currently working on, please visit the [roadmap](
 * [What is Camunda Cloud?](https://docs.camunda.io/docs/product-manuals/concepts/what-is-camunda-cloud)
 * [Getting Started Tutorial](https://docs.camunda.io/docs/guides/getting-started/create-camunda-cloud-account)
 * [Technical Concepts](https://docs.camunda.io/docs/product-manuals/zeebe/technical-concepts/index)
-* [BPMN Workflows](https://docs.camunda.io/docs/reference/bpmn-workflows/bpmn-primer)
+* [BPMN Processes](https://docs.camunda.io/docs/reference/bpmn-processes/bpmn-primer)
```

Comment:
The PR was merged yesterday to the next release branch https://github.com/camunda-cloud/camunda-cloud-documentation/pull/190, so it will not be visible soon, as we don't have development docs. If it's a big issue, we can patch the  URL in the latest docs already.